# Mass of the vehicle
mass: 1.5         # Kg
conf_mass: 10.0    # Confidence in the mass value, High value means high confidence

gravity: 9.81     # m/s2

# Inertia matrix of the vehicle
Ixx: 0.025        # Inertia along x -> The actual value (gazebo) is 0.029125
Ixy: 0.0          # Coupling between x and y 
Ixz: 0.0          # Coupling between x and z
Iyy: 0.025        # Inertia along y -> The actual value is 0.029125
Iyz: 0.0          # Coupling of rotation between y and z
Izz: 0.05         # Inertial along z -> The actual value is 0.055225
conf_I: 5       # Confidence in the Inertia matrix coefficient [0,1]

# A prior over the drag and thrust constants
Ct: 0.01           # Thrust coefficient -> This is actually going to be Ct * max_rotor_vel^2
conf_Ct: 0.0      # Confidence in the thurst coefficient

Cd: 0.01           # Drag coefficient -> This is actually going to be Cd * max_rotor_vel^2
conf_Cd: 0.0      # Confidence in the drag coefficient [0,1]

max_rotor_vel: 1100 # The maximum rotor velocity

# Enable or disable learning with gyroscopic effec
gyro_effect: False

# The motor model for fitting the function
motor_model: linear # {linear, quadratic, cubic, sigmoid_linear, sigmoid_quad}

# [TOD0]: Enable other configurations
# This assume X configuration
Dm_x: 0.13        # (m) Projection of the motor on the body x axis
Dm_y: 0.2         # (m) Projection of the motor on the body y axis
conf_Dm: 1.0      # Confidence in the projection of the motors on the axis [0,1]